### PR TITLE
Key path resolve

### PR DESCRIPTION
**Pull request by LuKks**

### CODE REVIEW

**Comment by LuKks** [comment]
@mafintosh Ready!

File: index.js [hunk 1]
```diff
@@ -466,8 +466,6 @@ function makeBee (key, corestore, onwait) {
 }
 
 function normalizePath (name) {
-  if (!name.startsWith('/')) {
-    return unixPathResolve(path.join('/', name))
-  }
-  return name
+  name = path.resolve('/', name)
+  return unixPathResolve(name)
```
Comment (mafintosh) [inline index.js:470]: this should just be, `unixPathResolve('/', name)`

Comment (mafintosh) [reply index.js:470]: @LuKks that's a feature

**Comment by LuKks** [comment]
@mafintosh Ready!

**Pull request merged**